docs/src/main/sphinx/connector/kafka.rst

```diff
@@ -302,9 +302,8 @@ this data must be mapped into columns to allow queries against the data.
     For textual topics that contain JSON data, it is entirely possible to not
     use any table definition files, but instead use the Trino
     :doc:`/functions/json` to parse the ``_message`` column which contains
-    the bytes mapped into an UTF-8 string. This is, however, pretty
-    cumbersome and makes it difficult to write SQL queries. This only works
-    when reading data.
+    the bytes mapped into an UTF-8 string. This is cumbersome and makes it
```

Comment:
an -> a? Other places do `a` here it's still `an`?

Reply:
No this is my bad. Will fix.